packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -1,17 +1,16 @@
 /*
- * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
```

Comment:
2015 because it's the same file in the diff? 2017 because that's when we wrote `Tooltip2`?

Reply:
yeah lol I'm not sure! probably 2015 cuz it's the same file.